Riot/Modules/Room/RoomViewController.m

```diff
@@ -6813,6 +6820,45 @@ - (void)showThreadsNotice
                                    completion:nil];
 }
 
+- (void)showThreadsBeta
```

Comment:
I know this is not a lot of code, but the `RoomViewController` is already so massive and every new bit makes that little bit worse. Could we extract this into some kind of `ThreadsBetaCoordinator`? Most of the code here is not directly related to the room, but rather the logic of enabling threads.

Reply:
Thanks for the suggestion, added.